src/data/restore.ts

```diff
@@ -116,6 +118,9 @@ const restoreElementWithProperties = <
     locked: element.locked ?? false,
   };
 
+  if ("subtype" in element && isValidSubtypeName(element.subtype, base.type)) {
```

Comment:
The check for `subtype  in element`  can be moved to `isValidSubtypeName` ? or may be better to rename it to `isValidSubtype` ?

Reply:
Putting the `"subtype" in element` check in `isValidSubtypeName` would require altering the implementation.  2a73ed1 may clarify why I am calling it `isValidSubtypeName` instead of `isValidSubtype`.  (See also the tests I included for `isValidSubtypeName` in 2a73ed1.)